### PR TITLE
add different flavor for etcd, masters and nodes

### PR DESCRIPTION
**Pull request by vgkowski**

### CODE REVIEW

File: platforms/openstack/neutron/variables.tf [hunk 1]
```diff
@@ -7,11 +7,27 @@ EOF
...
+
+variable "tectonic_openstack_worker_flavor_id" {
+  type = "string"
+
+  description = <<EOF
+The worker flavor ID as given in `openstack flavor list`. Specifies the size (CPU/Memory/Drive) of the VM.
+EOF
+}
+
+variable "tectonic_openstack_etcd_flavor_id" {
```
Comment (coreypobrien) [inline platforms/openstack/neutron/variables.tf]: Can we make `tectonic_openstack_etcd_flavor_id` optional when using self-hosted etcd (i.e. when `tectonic_experimental == true`)?

**Comment by lblackstone** [comment]
@vgkowski You'll need to run `make docs` and `make examples` to update related documentation/configs.

@s-urbaniak Functional changes LGTM

**Comment by lblackstone** [comment]
@vgkowski This PR will need a rebase due to https://github.com/coreos/tectonic-installer/pull/1325

Happy to take over this change if you don't want to.

**Comment by vgkowski** [comment]
Sorry not at work for 2 weeks. Please take it.

_________________________________________________________________________________________________________________________

Ce message et ses pieces jointes peuvent contenir des informations confidentielles ou privilegiees et ne doivent donc
pas etre diffuses, exploites ou copies sans autorisation. Si vous avez recu ce message par erreur, veuillez le signaler
a l'expediteur et le detruire ainsi que les pieces jointes. Les messages electroniques etant susceptibles d'alteration,
Orange decline toute responsabilite si ce message a ete altere, deforme ou falsifie. Merci.

This message and its attachments may contain confidential or privileged information that may be protected by law;
they should not be distributed, used or copied without authorisation.
If you have received this email in error, please notify the sender and delete this message and its attachments.
As emails may be altered, Orange is not liable for messages that have been modified, changed or falsified.
Thank you.

**Comment by lblackstone** [comment]
@s-urbaniak This can be closed now that #1344 has merged.

**Comment by s-urbaniak** [comment]
@lblackstone thanks for the hint, closing in favor of #1344

**Pull request closed without merging**